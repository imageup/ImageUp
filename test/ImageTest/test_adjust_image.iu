function main() -> int
{
        image: a = read("./images/google-chrome1.jpg");
        tuple: x = get_pixel(a, (0,1));
        prints("Change to pixel at (0, 1): ");
        prints("Before: " + StrParsef(x[0]) + " " + StrParsef(x[1]) + " " + StrParsef(x[2]));
        adjust_image(a, (0.2,3));
        tuple: y = get_pixel(a, (0,1));
        prints("After: " + StrParsef(y[0]) + " " + StrParsef(y[1]) + " " + StrParsef(y[2]));
		save("test.img1.jpg", a);
        return 0;
}

