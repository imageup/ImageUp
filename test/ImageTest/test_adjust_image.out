Change to pixel at (0, 1): 
Before: 255.000 255.000 255.000
After: 54.0000 54.0000 54.0000
